website/blog/2025-02-19-react-native-0.78.md

```diff
@@ -0,0 +1,217 @@
+---
+title: 'React Native 0.78 - React 19 and more'
+authors: [vonovak, shubham, fabriziocucci, cipolleschi]
+tags: [engineering]
+date: 2025-02-19
+---
+
+# React Native 0.78 - React 19 and more
+
+Today we are excited to release React Native 0.78!
+
+This release ships React 19 in React Native and some other relevant features like native support for Android Vector drawables and better brownfield integration for iOS.
+
+### Highlights
+
+- [React 19](#react-19)
+- [Towards smaller and faster releases](#towards-smaller-and-faster-releases)
+- [Opt-in for JavaScript logs in Metro](#opt-in-for-javascript-logs-in-metro)
+- [Better Android SVG support through Vector Drawable](#better-android-svg-support-through-vector-drawable)
+- [ReactNativeFactory on iOS](#reactnativefactory-on-ios)
```

Comment:
Note: All of these need prefixing with the final path (links are broken from the blog index page).

Reply:
These were working fine locally and they are working fine in the netlify preview... I don't see why they should break when we release the blogpost. 😅 

This is the same setup for the [0.77 blogpost](https://github.com/facebook/react-native-website/blob/main/website/blog/2025-01-21-version-0.77.md?plain=1#L16-L18) and they are working there.

Reply:
Those links are only visible if you click into the blog post because of the truncation block above them. But if you try clicking on any anchor links from the index page (reactnative.dev/blog), they're broken. They're also broken for some of the links in the 0.76 post too:
https://github.com/facebook/react-native-website/blob/2637718f90c931825ec3cca07a1b5be802c3838e/website/blog/2024-10-23-release-0.76-new-architecture.md?plain=1#L16

Reply:
ah, gotcha! thanks for the clarification!